aiowebostv/webos_client.py

```diff
@@ -114,7 +114,7 @@ async def connect_handler(self, res):
             if response["type"] == "hello":
                 self._hello_info = response["payload"]
             else:
-                raise PyLGTVPairException("Unable to say hello")
+                raise WebOsTvCmdException(f"Invalid request type {response}")
```

Comment:
This was changed by purpose, even if TV is not paired it should answer to hello request, since `WebOsTvPairException` is used to indicate that the TV is not paired (asking user to go through pairing process) we should not raise pairing error here.